contrib/verify-commits/README.md

```diff
@@ -33,6 +33,17 @@ Configuration files
 * `trusted-keys`: This file should contain a \n-delimited list of all PGP fingerprints of authorized commit signers (primary, not subkeys).
 * `allow-revsig-commits`: This file should contain a \n-delimited list of git commit hashes. See next section for more info.
 
+Import trusted keys
+-------------------
+In order to check the commit signatures you must add the trusted PGP keys to your machine. This can be done in Linux by running
+    while read -r LINE; do gpg --recv-keys $LINE; done < contrib/verify-commits/trusted-keys
```

Comment:
Should be a code block? :-)

Reply:
I changed it to a code block (and the other indented command is now also a code block)
I also switched it to use the simplified command suggested by @dongcarl